### DISCUSSION TITLE
Is there a way to update a record and retrieve its old value?

### DISCUSSION THREAD

**Discussion by euberdeveloper** [prisma/prisma#6209] [Q&A] [answered]
If I update a record and I want to know its old value, now, I should firstly query the record and then do the update.

Is there an option that says to the update function to retrieve the old record value?

**Comment by ryands17** [comment, marked as answer]
@euberdeveloper 👋 

Currently you would need a separate query to fetch the old value first as `update` will give you thr values after it has been updated.

**Reply by euberdeveloper** [reply]
I think that a feature request could be done, thank you for your answer

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
